pcdsdevices/mv_interface.py

```diff
@@ -252,14 +288,23 @@ def sync(self):
         logger.debug('call %s presets.sync()', self._device.name)
         self._remove_methods()
         self._cache = {}
+        logger.debug('filling %s cache', self.name)
         for preset_type in self._paths.keys():
-            try:
-                self._cache[preset_type] = self._read(preset_type)
-            except FileNotFoundError:
+            path = self._path(preset_type)
+            if path.exists():
+                try:
+                    self._cache[preset_type] = self._read(preset_type)
+                except OSError:
+                    self._log_flock_error()
+            else:
                 logger.debug('No %s preset file for %s',
                              preset_type, self._device.name)
         self._create_methods()
 
+    def _log_flock_error(self):
+        logger.error('Cannot acquire file lock for %s', self.name)
+        logger.debug('', exc_info=True)
```

Comment:
Might encourage a more novice friendly error message. `"File is currently being edited, can not acquire filelock"`

Reply:
Your suggestion isn't necessarily true. Being unable to acquire a file lock can also be because something has gone very wrong.

Reply:
`"Unable to acquire filelock. File may be being edited by another user"`